notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1291 @@
+{
```

Comment:
<ol><li>that will allow you &gt; that allows you</li><li>Do we want to mention data aggregation by time series ID prior to training?</li><li>In our meeting notes, you provided a detailed description of how we iterate over the time series, perform hyperparameter tuning, etc. Is this something we should include here? Would the users want to know this and is it something we can expose to them? At the very least, I think we should repeat what we said in the introduction: "The training pipeline performs hyperparameter tuning using <a href="https://en.wikipedia.org/wiki/Hyperparameter_optimization#Grid_search" target="_blank">grid search</a> and Prophet's built-in backtesting logic."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>I talk about it more at the beginning of this section.</p>